rest-api/config/config_stable.json

```diff
@@ -36,6 +36,13 @@
         "whitelisted_ip_ranges" : {
           "ip4": ["10.50.178.54"]
         }
+    },
+    "awardee-ucirvine@all-of-us-ops-data-api-stable.iam.gserviceaccount.com": {
+      "roles": ["awardee_sa"],
+      "awardee": "CAL_PMC",
+      "whitelisted_ip_ranges" : {
+        "ip4": ["10.50.178.54"]
```

Comment:
This doesn't match the ticket

Reply:
weird. Ok, fixed. Good catch. CI builds still failing though.